### PR TITLE
docs(Observable): add documentation for interval

### LINKED ISSUE #80: Add docs for operator : creation - interval
Add docs for operator interval under group creation 
 
Below is minimum details needed 
 
- [ ] name 
- [ ] operatorType 
- [ ] signature 
- [ ] parameters 
- [ ] short description 
- [ ] Technical description 
- [ ] Visual example  (Marble diagram) 
- [ ] Real world example (Non-framework) 
- [ ] Related Operators

### PR DESCRIPTION
**Pull request by cschomer**
Add docs for operator : creation - interval 

Close #80

### CODE REVIEW

**Comment by JWO719** [comment]
Could you change your commit messages. There are several :, these make travis break. If you need any help, just inform me. Thanks for your work!!!

**Comment by cschomer** [comment]
Yes sorry about that, should be corrected and passing now.

**Review by ashwin-sureshkumar — CHANGES_REQUESTED** [review]
Thanks for the PR. Please address requested changes.

File: src/operator-docs/creation/interval.ts [hunk 1]
```diff
@@ -1,6 +1,52 @@
...
+          import { take } from 'rxjs/internal/operators/take';
+          const observable = interval(1000).pipe(take(5));
+          const subscription = observable.subscribe({
+            next: (n) => console.log(n), // triggered every 1000ms from interval
+            complete: () => console.log('complete'), // logs 'complete'
+          });
+      `,
+      externalLink: {
+        platform: 'JSBin',
+        url: 'http://jsbin.com/gaxinafovo/1/edit?html,js,console,output'
```
[the remark this answers is not in the document]
Comment (cschomer) [reply src/operator-docs/creation/interval.ts]: Updated path to use embed

File: src/operator-docs/creation/interval.ts [hunk 2]
```diff
@@ -1,6 +1,52 @@
...
+  },
+  walkthrough: {
+    description: `Creates an Observable that emits sequential numbers every specified
+     interval of time, on a specified IScheduler`
+  },
+  examples: [
+    {
+      name: 'Observable emits ascending numbers, one every second (1000ms)',
+      code: `
+          import { interval } from 'rxjs/internal/observable/interval';
```
[the remark this answers is not in the document]
Comment (cschomer) [reply src/operator-docs/creation/interval.ts]: Changed path to correct path

**Comment by ladyleet** [comment]
@cschomer have time to make these changes yet? we'd love to merge!

**Comment by cschomer** [comment]
@ladyleet Hi I checked in 2 commits for the requested changes, are those not showing up? I think with those two that should address the two requested changes.

[commit ea9c472d97e6c7b0f00feabc55bed642ab0781d1](https://github.com/ReactiveX/rxjs-docs/pull/296/files/fde5aa85997d5986a42f26ea0521969f450a2aa9)

**Review by ashwin-sureshkumar — APPROVED** [review]

**Comment by ashwin-sureshkumar** [comment]
@cschomer - Thanks for the PR. 

Please use `Close #<issue-number>` in your future commits in the description. This will auto-close the issue once the PR is merged in. 

For this one, we can manually close the issue.

File: src/operator-docs/creation/interval.ts [hunk 3]
```diff
@@ -1,6 +1,52 @@
...
+  walkthrough: {
+    description: `Creates an Observable that emits sequential numbers every specified
+     interval of time, on a specified IScheduler`
+  },
+  examples: [
+    {
+      name: 'Observable emits ascending numbers, one every second (1000ms)',
+      code: `
+          import { interval } from 'rxjs/observable/interval'
+          import { pipe } from 'rxjs';
```
Comment (JWO719) [inline src/operator-docs/creation/interval.ts]: pipe doesn't need to be imported, I think :D

**Review by niklas-wortmann — CHANGES_REQUESTED** [review]
Just a little change. After that one, I think it can be merged.

**Pull request closed without merging**